pkg/gofr/gofr.go

```diff
@@ -398,3 +398,7 @@ func (a *App) AddCronJob(schedule, jobName string, job CronFunc) {
 		a.Logger().Errorf("error adding cron job, err : %v", err)
 	}
 }
+
+func (a *App) UseCassandra(db datasource.Cassandra) {
```

Comment:
seems like datasource is not imported here.
Also we can move this to externalDB file in the same pacakge

Reply:
I will make the changes